packages/core/src/components/overlay/overlay.tsx

```diff
@@ -69,6 +69,13 @@ export interface IOverlayableProps extends IOverlayLifecycleProps {
      */
     usePortal?: boolean;
 
+    /**
+     * The container that the overlay renders its contents to.
+     * This prop only available when `usePortal` is set to `true`
+     * @default document.body
+     */
+    container?: HTMLElement;
```

Comment:
this name is not sufficiently clear. two options on the API here:
1. `portalContainer` to make it obvious how this prop is used.
2. `portalProps` to support full portal customization (per #2977).

do you have a preference?

Reply:
I'd like to change it to `portalContainer`